src/directives/unsafe-svg.ts

```diff
@@ -48,9 +48,11 @@ export const unsafeSVG = directive((value: unknown) => (part: Part): void => {
   }
 
   const template = document.createElement('template');
-  template.innerHTML = `<svg>${value}</svg>`;
+  const svgElement =
+      document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+  template.content.appendChild(svgElement);
+  svgElement.innerHTML = value as string;
```

Comment:
I'd still use `template.innerHTML` so that the element doesn't change documents:

```suggestion
  template.innerHTML = `<svg></svg>`;
  template.content.firstChild.innerHTML = value as string;
```

Reply:
We can't assign a bare string to template.innerHTML, it must go through a policy.

We'd either have to export the policy from template-result (which is a security issue) or create a new policy for unsafeSVG

Reply:
ack. Maybe consider `t.content.ownerDocument`? Not a big deal though.

Reply:
+1, done